Libraries/LibJS/Runtime/JSONObject.cpp

```diff
@@ -495,4 +503,60 @@ ThrowCompletionOr<Value> JSONObject::internalize_json_property(VM& vm, Object* h
     return TRY(call(vm, reviver, holder, PrimitiveString::create(vm, name.to_string()), value));
 }
 
+// https://tc39.es/proposal-json-parse-with-source/#sec-json.rawjson
+JS_DEFINE_NATIVE_FUNCTION(JSONObject::raw_json)
+{
+    auto& realm = *vm.current_realm();
+
+    // 1. Let jsonString be ? ToString(text).
+    auto json_string = TRY(vm.argument(0).to_string(vm));
+
+    // 2. Throw a SyntaxError exception if jsonString is the empty String, or if either the
+    // first or last code unit of jsonString is any of 0x0009 (CHARACTER TABULATION),
+    // 0x000A (LINE FEED), 0x000D (CARRIAGE RETURN), or 0x0020 (SPACE).
```

Comment:
When we wrap spec comments, we hang the indentation. And we wrap loosely around 120 characters:
```c++
// 2. Throw a SyntaxError exception if jsonString is the empty String, or if either the first or last code unit of
//    jsonString is any of 0x0009 (CHARACTER TABULATION), 0x000A (LINE FEED), 0x000D (CARRIAGE RETURN), or
//    0x0020 (SPACE).
```